WcaOnRails/config/environments/development.rb

```diff
@@ -52,9 +52,9 @@
   config.assets.raise_runtime_errors = true
 
   # One can activate/test the fallback to English using :
-  # config.i18n.fallbacks = true
+  config.i18n.fallbacks = true
```

Comment:
Seems like there was a bug before. I support making this change here and now, though.